browser/app/js/buckets/BucketList.js

```diff
@@ -17,14 +17,29 @@
 import React from "react"
 import { connect } from "react-redux"
 import { Scrollbars } from "react-custom-scrollbars"
+import InfiniteScroll from "react-infinite-scroller"
 import * as actionsBuckets from "./actions"
-import { getVisibleBuckets } from "./selectors"
+import { getFilteredBuckets } from "./selectors"
 import BucketContainer from "./BucketContainer"
 import web from "../web"
 import history from "../history"
 import { pathSlice } from "../utils"
 
 export class BucketList extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      page: 1
+    }
+    this.loadNextPage = this.loadNextPage.bind(this)
+  }
+  componentWillReceiveProps(nexProps) {
```

Comment:
Very minor issue here, can we rename `nexProps` here to `nextProps` in terms of consistency with the rest of the app?

Also, as a future consideration, we would need to eventually move on from using `componentWillReceiveProps` method as it will be deprecated after React 17. For now, we can take this in and think about a different approach in the future.

Reply:
@kaankabalak thanks for catching this, i missed it. I will move to use componentDidUpdate.

Reply:
@kaankabalak done, please review again